src/nupic/ntypes/Dimensions.hpp

```diff
@@ -83,24 +83,24 @@ class Dimensions : public std::vector<UInt> {
 
   /**
    * Create a new Dimensions object.
+   * The dimension in index 0 is the one that moves fastest while iterating.
+   * in 2D coordinates, x,y; the x is dimension[0], y is dimension[1].
```

Comment:
what does it mean "moves fastest"? Can you give example d[1][3] = ??

Reply:
In a contiguous block of memory, elements are with the first dimension (the left most), then increment the second, etc toward the right.   example d[2][3]   {{ 1, 1}, {2 , 2}, {3,3}};
would appear in contiguous memory as:
``` 
1, 1,  2,  2 ,  3, 3
```
So the first index is moving the fastest as you walk the array.